### PR TITLE
Address book

### PR DESCRIPTION
**Pull request by sereda1997**
-Added address book tests 
-Allure reports in dependencies
-Added TestRunnerYura

### CODE REVIEW

File: OpencartTest/src/com/opencart/tools/TestRunnerYura.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import com.opencart.pages.account.MyAccountPage;
+import com.opencart.pages.cart.CartPage;
+import com.opencart.pages.wishlist.WishListPage;
+import org.testng.ITestResult;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+
+import org.testng.annotations.BeforeClass;
+
+public class TestRunnerYura {
```
Comment (VitaliiPopov) [inline OpencartTest/src/com/opencart/tools/TestRunnerYura.java]: Perhaps "AddressPageTestRunner" will be better?

File: OpencartTest/src/com/opencart/tools/TestRunnerYura.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    @AfterClass
+    public void afterClass() {
+        MyAccountPage myAccountPage = getHomePage().clickMyAccountInDropdown();
+        AddressBookPage addressBookPage = myAccountPage.clickModifyYourAddressBookEntries();
+        addressBookPage.deleteAll();
+        Driver.quit();
+    }
+
+
```
Comment (VitaliiPopov) [inline OpencartTest/src/com/opencart/tools/TestRunnerYura.java]: Empty space

File: OpencartTest/src/com/opencart/tools/Utility.java [hunk 3]
```diff
@@ -14,7 +14,7 @@ public static String getScreenshot(WebDriver driver) {
         TakesScreenshot takesScreenshot = (TakesScreenshot) driver;
 
         File src = takesScreenshot.getScreenshotAs(OutputType.FILE);
-        String path = "/Screenshot/" + System.currentTimeMillis() + ".png";
+        String path = "C:\\OpenCartMerged\\OpencartTest\\src" + System.currentTimeMillis() + ".png";
```
Comment (ostret) [inline OpencartTest/src/com/opencart/tools/Utility.java]: This will fail on Jenkins

File: OpencartTest/src/test/AddressTests.java [hunk 4]
```diff
@@ -34,35 +27,34 @@ public void abilityToAddAddress(String myAccountDropdownText) throws Interrupted
...
 
-    @Test(priority = 3)
+    @Test(priority = 3,description = "Check the ability to edit address")
     public void editAddress() throws InterruptedException {
-        MyAccountPage myAccountPage = getHomePage().clickMyAccauntInDropdownHardcode();
+        MyAccountPage myAccountPage = getHomePage().clickMyAccountInDropdown();
         AddressBookPage addressBookPage = myAccountPage.clickModifyYourAddressBookEntries();
         EditAdressPage edittest = addressBookPage.clickEditAddresss();
-        AddressBookPage adres = edittest.edituser(jsonDataConfig.getFirstNameFromJson(3),
+        AddressBookPage adres = edittest.editUser(jsonDataConfig.getFirstNameFromJson(3),
```
Comment (ostret) [inline OpencartTest/src/test/AddressTests.java]: i still think this is extremely incomprehensible for anyone except author

**Pull request merged**